src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers, createStore } from 'redux';
+import * as bookReducers from './books/books';
+import * as categoryReducers from './categories/categories';
+
+const rootReducer = combineReducers({
+  books: bookReducers,
+  categories: categoryReducers,
+});
+
+const store = createStore(rootReducer);
+
+export default store;
```

Comment:
- [ ] Please check this console error and please try to fix it
![Screenshot from 2022-03-28 19-30-37](https://user-images.githubusercontent.com/62705947/160454724-cb30b90b-aa59-45c8-b13a-87d5d261d0c7.png)

Reply:
@GzimAsani thanks for catching that I fixed it!